.env.example

```diff
@@ -1,4 +1,4 @@
 SITE_URL=http://localhost:3000
 NEXT_PUBLIC_WOO_SITE_URL=http://yourwocommercesite.com
 WOO_CONSUMER_KEY=xxxxx
-WOO_SECRET=xxxxx
\ No newline at end of file
+WOO_SECRET=xxxxx
```

Comment:
I am not sure if we are using these env vars somewhere. Can we remove them safely?

Reply:
@thelovekesh If its not used. You can remove it please.